src/services/user/model.js

```diff
@@ -14,5 +14,10 @@ const userSchema = new Schema({
 	birthday: {type: Date}
 });
 
+userSchema.methods.getPermissions = function() {
+	const roleModel = require('../role/model');
```

Comment:
Well, turns out there's a connection between users and roles. Honestly, how else would you do this? Please be a bit more professional and constructive with your comments.